CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 * Add `ProfilesLabel` and `SettingsLabel` components for second settings pane. UIDEXP-39.
 * Add mapping profiles pane component to display static data. UIDEXP-41.
 * Handle "form elements must have labels" accessibility problem for the file uploader form. STDTC-7.
+* Ensure that jobs lists are structured correctly accessibility problem. UIDEXP-49.
```

Comment:
I think, "Update jobs list structure according to the accessibility requirements" or something like this is more accurate

Reply:
Will change, thank you.